addons/mail/static/src/webclient/commands/mail_providers.js

```diff
@@ -7,7 +7,7 @@ const { Component, xml } = owl;
 
 class DialogCommand extends Component {}
 DialogCommand.template = xml`
-    <div class="o_command_default">
+    <div class="o_command_default d-flex flex-grow-1 align-items-center justify-content-between px-4 py-2 o_cursor_pointer">
```

Comment:
Not blocking but just a remark: there seems to be a lot of repetition with this part, shouldn't it be its own component?

Reply:
@seb-odoo If you mean repetition in the DOM, I see you point... But this is the guideline that was decided for the SCSS cleanup/revamp.
On regard to the code base, there is no repetition as this is in a single OWL component's template.

Reply:
Indeed it's fine and not actually repetition if we consider each component individually, but it gets annoying to maintain if in practice we need to update all of them whenever there is a change on the general style of the commands.

It just depends where we set the limit to what is a unique component, I'm fine either way, I was just mentioning it to be sure. It's a good approach to consider them individual and accept some repetition for now.
And only make a common component later if we notice they always get updated together.